xray/commands/audit/sca/java/mvn.go

```diff
@@ -1,98 +1,146 @@
 package java
 
 import (
+	_ "embed"
 	"errors"
 	"fmt"
-	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
-	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
-	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
-	mvnutils "github.com/jfrog/jfrog-cli-core/v2/utils/mvn"
-	"github.com/jfrog/jfrog-client-go/auth"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
 	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
 	"github.com/jfrog/jfrog-client-go/utils/log"
 	xrayUtils "github.com/jfrog/jfrog-client-go/xray/services/utils"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
 )
 
-func buildMvnDependencyTree(params *DependencyTreeParams) (modules []*xrayUtils.GraphNode, uniqueDeps []string, err error) {
-	buildConfiguration, cleanBuild := createBuildConfiguration("audit-mvn")
-	defer func() {
-		err = errors.Join(err, cleanBuild())
-	}()
+const (
+	mavenDepTreeJarFile    = "maven-dep-tree.jar"
+	mavenDepTreeOutputFile = "mavendeptree.out"
+	mavenDepTreeVersion    = "1.0.2"
+	TreeCmd                = "tree"
+	ProjectsCmd            = "projects"
+	settingsXmlFile        = "settings.xml"
+)
+
+var settingsXmlTemplate = `<?xml version="1.0" encoding="UTF-8"?>
+<settings xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.2.0 http://maven.apache.org/xsd/settings-1.2.0.xsd" xmlns="http://maven.apache.org/SETTINGS/1.2.0"
```

Comment:
Let's embed the settings.xml from a formatted XML file to a string variable. See this for example:
https://github.com/jfrog/build-info-go/blob/v1.9.15/build/gradle.go#L31